IpmiFeaturePkg/SpmiTable/SpmiTable.c

```diff
@@ -0,0 +1,205 @@
+/** @file
+  SPMI Table installation.
+
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiDxe.h>
+
+#include <IndustryStandard/Acpi.h>
+#include <IndustryStandard/ServiceProcessorManagementInterfaceTable.h>
+#include <IndustryStandard/IpmiNetFnApp.h>
+#include <Protocol/AcpiSystemDescriptionTable.h>
+#include <Protocol/AcpiTable.h>
+#include <Protocol/IpmiTransportProtocol.h>
+
+#include <Library/BaseLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/UefiLib.h>
+#include <Library/PcdLib.h>
+
+// Service Processor Management Interface Table definition
+EFI_ACPI_SERVICE_PROCESSOR_MANAGEMENT_INTERFACE_TABLE  gSpmiTable = {

```

Comment:
```suggestion
STATIC EFI_ACPI_SERVICE_PROCESSOR_MANAGEMENT_INTERFACE_TABLE  mSpmiTable = {
```
Module level global variables are supposed to use the `m` prefix